code/modules/cargo/packs.dm

```diff
@@ -829,6 +858,24 @@ datum/supply_pack
 	crate_type = /obj/structure/closet/crate/secure
 	dangerous = TRUE
 
+/datum/supply_pack/science/bz_canister
+	name = "BZ Canister"
+	cost = 4000
+	access = access_rd
+	contains = list(/obj/machinery/portable_atmospherics/canister/bz)
+	crate_name = "bz canister crate"
+	crate_type = /obj/structure/closet/crate/secure
+	dangerous = TRUE
+
+/datum/supply_pack/science/freon_canister
+	name = "Freon Canister"
+	cost = 6000
+	access = access_rd
```

Comment:
Can crates have more than one access type? CE (Atmos) and RD (Toxins) both have uses for them. If this is not the case, it makes more sense for CE to have them rather than the RD.